world-bank-app/src/App.js

```diff
@@ -22,9 +29,21 @@ class App extends React.Component {
     this.setState({ results: data ? [...data] : [] });
   };
 
-  logIn(email) {
-    this.setState({ loggedIn: true, user: email });
-  }
+  logIn = () => {
+    const { cookies } = this.props;
+
+    const currentState = this.state.isLoggedIn;
+    if (this.state.isLoggedIn) {
+      cookies.remove("sessionId");
+      cookies.remove("user_id");
+    }
+    console.log(cookies.getAll());
+    this.setState({ isLoggedIn: !currentState });
+  };
+
```

Comment:
Really great, can pass this down everywhere and add it to the onClick of log out buttons